### ISSUE TITLE
Hardware Decoding for Flatpak Apps seems to still be broken for some Apps

### ISSUE THREAD

**Issue by DarkSide1305** [ValveSoftware/SteamOS#1037]
#### Your system information

* Steam client version: 1682708537
* SteamOS version: 3.4.6
* Opted into Steam client beta?: No 
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:

In recent Issue #1014 it was reported that a recent Flatpak update broke Hardware Decoding for Flatpak Apps. Going through the now closed issue it appears it should have been fixed by org.kde.Platform version 5.15-22.08 and 6.4. I have checked my Flatpak list, and indeed my branch are those 2 exact versions. But trying to use Steam Remote Play via the Flatpak version of RetroArch (and a lot of other Emulators like Ryujinx, RPCS3, PCSX2, Dolphin all via Flatpak) has resulted in working Audio but infinite loading and Blackscreen for Video. Using the AppImage version of RetroArch for Steam Remote Play Together works perfectly without issue. This issue is not SteamOS directly, but as lostgoat mentioned in that issue these problems are useful to have in the tracker, and if this issue happens again to open a new issue, so I am opening this issue. I am not 100% sure this is the exact same issue as in #1014, and if it is not this Issue can be closed as im not sure if this is related to issue #1014 or if this is the fault of the RetroArch Flatpak (but considering how many other Emulators dont work either I would assume this is not the fault of RetroArch), but it seems very very similar at least.

#### Steps for reproducing this issue:

1. Download the Flatpak version of RetroArch (or other Emulators mentioned above)
2. Try to stream them via Remote Play Together on a different PC

**Comment by DarkSide1305** [comment]
Heya. Yeah I wasn't too sure if to report this here or not, id gladly report this to the mesa Flatpak maintainers, but would you maybe be able to link me to where I would need to report this? Im having a bit of a hard time finding their Flatpak Repository.

**Comment by DarkSide1305** [comment]
I have looked around even more the last few days, joined an IRC and I found a few more things. Apparently the issue above comes because mesa was asked by Redhat to disable all patent ecumbered codecs, thats why they are disabled. But I found a post from about 5 months ago on Reddit (this was on r/Fedora, but its quite old so I thought id share my findings anyway) that Mesa 22.2.4 solves the patent encumbered codecs, because with it The Mesa (Extra) runtime has those codecs enabled by default, and apparently all Apps with a base runtime of 22.08 should make use of this newer Mesa update. Now, if im completely honest I dont have much knowledge about all of this, im quite new to Linux and the only thing I can find about all of this is just piece meal from here and there, but I thought id share my findings in this issue anyway, maybe it'll help.
